docs/Theming-Fonts-And-Typography.md

```diff
@@ -0,0 +1,14 @@
+# Fonts & Typography
```

Comment:
where do you think we should add documentation about adding fonts? Theres a whole bunch of inline comments I removed about this but we should definitely bring that back in the readme somewhere. This involves example of adding expo fonts and non-expo font files.

Reply:
I think that would fit in here. Unfortunately I don't have the knowledge on how custom fonts are added, what expo-fonts supports, etc.

Reply:
I can work on that (wont be today though).

Reply:
I'll leave this for a future PR.